src/css/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$primary-text: #fff;
+$secondary-text: #F4A460; // currently just used for visited links in errors and such.
+
+$primary-bg: #000;
```

Comment:
we should do something like:
```sass
// colors.scss
$black: #000

//vars.scss
$primary-bg: $black

// other files
background-color: $primary-bg
```

that way, if we feel like changing our "black" to be `#3c3c3c` or something, we only change it in one place. The primary/stuff is already good the way it is :)